docs/guides/Redux-Integration.md

```diff
@@ -44,3 +44,13 @@ class App extends React.Component {
 Now, your navigation state is stored with redux, and you can fire navigation actions using redux.
 
 When a navigator is given a `navigation` prop, it relinquishes control of the state. So you are now responsible for persisting state, handling deep linking, integrating the back button, etc.
+
+When you are nesting navigators, the navigation state is automatically passed down as long as a navigator is attached as a screen. For example:
+
+```js
+StackNavigator({
+  Home: { screen: TabNavigator },
+});
+```
+
+When you attach `StackNavigator` to Redux, `TabNavigator` automatically gets attached and you don't have to attach it separately.
```

Comment:
Again in here it's hard for me to understand what does it mean: to attach `StackNavigator` to Redux?

This last sentence is very unclear to me.